e2e-tests/IIoTPlatform-E2E-Tests/deploy/IoTHubLegacyPublisherDeployments.cs

```diff
@@ -0,0 +1,87 @@
+﻿// ------------------------------------------------------------
+//  Copyright (c) Microsoft Corporation.  All rights reserved.
+//  Licensed under the MIT License (MIT). See License.txt in the repo root for license information.
+// ------------------------------------------------------------
+
+namespace IIoTPlatform_E2E_Tests.Deploy {
+    using System;
+    using System.Collections.Generic;
+    using Newtonsoft.Json;
+    using TestExtensions;
+
+    public sealed class IoTHubLegacyPublisherDeployments : DeploymentConfiguration {
+
+        /// <summary>
+        /// MessagingMode that will be used for configuration of OPC Publisher.
+        /// </summary>
+        public readonly MessagingMode MessagingMode;
```

Comment:
`MessagingMode` does not seem to be used. In the case of `2.8.x` publisher it makes sense since we support for `PubSub` and `Samples` modes. Do we have anything similar for `2.5` ? If no, can we remove it then ?

Reply:
removed we don't need for publisher 2.5